test/lib/builder/builder.js

```diff
@@ -84,7 +84,7 @@ test("Build application.a", (t) => {
 	return builder.build({
 		tree: applicationATree,
 		destPath,
-		excludedTasks: ["generateComponentPreload", "generateStandaloneAppBundle", "generateVersionInfo"]
+		excludedTasks: ["escapeNonAsciiCharacters", "generateComponentPreload", "generateStandaloneAppBundle", "generateVersionInfo"]
```

Comment:
Instead of excluding the task everywhere, please rather add the necessary `"propertiesFileEncoding": "ISO-8859-1"` configuration to the project trees where necessary.

This is a consequence of our decision to make the defaulting in the type-formatters instead of the -builders. Others, that use our `build()` API directly, need to do the same (and not exclude the task).

Reply:
done